src/ResourceManager/KeyVault/Commands.KeyVault/packages.config

```diff
@@ -1,11 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.ActiveDirectory.GraphClient" version="2.1.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.KeyVault" version="2.3.2" targetFramework="net452" />
-  <package id="Microsoft.Azure.KeyVault.WebKey" version="2.0.7" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.KeyVault" version="2.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.KeyVault" version="3.0.0-alpha" targetFramework="net452" />
+  <package id="Microsoft.Azure.KeyVault.WebKey" version="3.0.0-alpha" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.KeyVault" version="2.4.0-alpha" targetFramework="net452" />
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net452" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
```

Comment:
Please regenerate the help files: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md

Reply:
Following up with a separate commit for this.